tda/debug.py

```diff
@@ -52,7 +56,7 @@ def register_redactions_from_response(resp):
     if resp.ok:
         try:
             register_redactions(resp.json())
-        except json.decoder.JSONDecodeError:
+        except JSONDecodeError:
```

Comment:
So my concern here is that users who mix JSON libraries will encounter this issue. How about attempting to import both exceptions, renaming the `simplejson` one to the builtin one if it's not available, and then `except`'ing on both? 

That way if `simplejson` is available we end up testing for both but if only the builtin one is available then we just check for it twice.

Reply:
This is really a `requests` library problem, not tda-api. Personally I think they shouldn't have left their API so conditional upon the JSON library they happen to import. 

That said, the best thing that tda-api can do is know exactly what they are importing. That code is here:
https://github.com/psf/requests/blob/master/requests/compat.py#L28
Users can't "mix" JSON libraries as you say, because they don't get to choose which library gets used by `requests`.

Reply:
True, but my concern is that your change assumes knowledge of the behavior of the `requests` library, and while this approach is currently appropriate, I suggest we can futureproof ourselves fully by being redundant in what we check for.

Reply:
Others on the net have suggested to just use ValueError which catches both. Thoughts on that?

Reply:
Strongly disagree, exceptions should be as specific as possible.

Reply:
Ok, I'll make the change an update the PR